core/src/test/scala/integration/kafka/api/QuotasTest.scala

```diff
@@ -15,12 +15,13 @@
 package kafka.api
```

Comment:
I tried to reproduce but could not. Anyway, I found another build that had a failure:
```
kafka.api.QuotasTest > testThrottledProducerConsumer FAILED
    junit.framework.AssertionFailedError: Should have been throttled
        at junit.framework.Assert.fail(Assert.java:47)
        at junit.framework.Assert.assertTrue(Assert.java:20)
        at kafka.api.QuotasTest.testThrottledProducerConsumer(QuotasTest.scala:136)
```